src/main/java/by/exadel/internship/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,17 @@
+package by.exadel.internship.exception;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class UserNotFoundException extends RuntimeException {
```

Comment:
As far as I understand we want to use not specific exception
```suggestion
public class NotFoundException extends RuntimeException {
```

Reply:
done